witchcraft-logging-idea/src/test/java/com/palantir/witchcraft/java/logging/idea/WitchcraftLogFormatFilterTest.java

```diff
@@ -54,7 +54,8 @@
             + "\"traceId\":\"ba3200b6eb01999a\",\"unsafeParams\":{\"path\":\"/api/sleep/10\","
             + "\"millis\":\"10\"}}";
     private static final String REQUEST_FORMATTED =
-            "[2019-05-24T16:40:36.703Z] \"GET /api/sleep/10 HTTP/1.1\" " + "503 78 1935";
+            "[2019-05-24T16:40:36.703Z] \"GET /api/sleep/10 HTTP/1.1\" 503 78 1935 μs (host: localhost:8443, "
+                    + "connection: Keep-Alive, accept-encoding: gzip, user-agent: okhttp/3.13.1)";
```

Comment:
I wonder if this is going to be very spammy? I can see value of the host/user agent as you an see where the request came from (including in dev-env tests), but do we think `connection`/other parameters are important for daily debugging use?

Reply:
iirc dev-env removes the `_sampled` parameter from service logs, so I could imagine we may want to strip it here as well. In the general case though, the data that's spammy here also spams our indexers, and may be worthwhile to remove from our request logging infra if we don't need it. What do you think?

Reply:
Good point, we can filter there if we need to.